### PR TITLE
layers: Remove UNASSIGNED for non-errors

### PR DESCRIPTION
**Pull request by spencer-lunarg**
In effort to clean up the remaining `UNASSIGNED` VUs, I have gone and removed anything that is not an error from having UNASSIGNED (except for Best Practice, which is its own project in itself)

I few it has there are 3 prefixes

1. `VUID-` which is reserved from proper VUs in the spec
2. `UNASSIGNED-` which are errors that don't have a VU (for one reason or another)
3. everything

Things such as warnings will never have a `VUID` label, so it is not actually unassigned a VUID

### CODE REVIEW

File: layers/object_tracker/object_lifetime_validation.h [hunk 1]
```diff
@@ -88,7 +81,7 @@ class ObjectLifetimes : public ValidationObject {
         if (!inserted) {
             // The object should not already exist. If we couldn't add it to the map, there was probably
             // a race condition in the app. Report an error and move on.
-            (void)LogError(kVUID_ObjectTracker_Info, object, loc,
+            (void)LogError("UNASSIGNED-ObjectTracker-Insert", object, loc,
```
Comment (artem-lunarg) [inline layers/object_tracker/object_lifetime_validation.h:84]: I have impression that usually VUIDs are not assigned for race conditions. I remember Tobias said something related o gitlab (https://gitlab.khronos.org/vulkan/vulkan/-/issues/3616#note_429860). I explained this to myself that there could be many source of race condition that are impractical/impossible to enumerate all combinations, but not 100% sure.

**Review by artem-lunarg — APPROVED** [review]
LGTM. One comment about UNASSIGNED/VUID for race conditions.

File: layers/core_checks/cc_image.cpp [hunk 2]
```diff
@@ -308,7 +308,7 @@ bool CoreChecks::PreCallValidateCreateImage(VkDevice device, const VkImageCreate
             total_size = (total_size + ig_mask) & ~ig_mask;
 
             if (total_size > format_limits.maxResourceSize) {
-                skip |= LogWarning(kVUID_Core_Image_InvalidFormatLimitsViolation, device, error_obj.location,
+                skip |= LogWarning("WARNING-Image-InvalidFormatLimitsViolation", device, error_obj.location,
```
[the remark this answers is not in the document]
Comment (spencer-lunarg) [reply layers/core_checks/cc_image.cpp:311]: if we add `ERROR` now, then the message will print both `Warning` and `Error` which is even more confusing

I am not against moving this to an error, but then it needs to be `LogError` and it is not a VU, it needs an explanation comment why

File: layers/object_tracker/object_lifetime_validation.h (hunk 1 above)
Comment (spencer-lunarg) [reply layers/object_tracker/object_lifetime_validation.h:84]: Will look into this, for now, this is just moving the string here, but agree, maybe it shouldn't be

Comment (spencer-lunarg) [reply layers/object_tracker/object_lifetime_validation.h:84]: added a todo comment

**Review by arno-lunarg — APPROVED** [review]

**Pull request merged**